Dockerfile

```diff
@@ -17,6 +17,7 @@ COPY --from=builder /usr/src/app/node_modules /usr/src/app/node_modules
 COPY package.json /usr/src/app
 COPY index.js /usr/src/app
 RUN mkdir ~/.ssh
-RUN ssh-keyscan -H wetty-ssh >> ~/.ssh/known_hosts
+RUN echo 'ssh-keyscan -H ${SSHHOST} >> ~/.ssh/known_hosts' > start_script.sh
+RUN echo 'yarn start' >> start_script.sh
```

Comment:
using a  startup script breaks the ability to add commands args

Reply:
I was thinking args can be added in the dockerfile itself. 
Had to do it like this as auto login fails if the host key is not verified. Do you have a better way?
Also, added the SSH Host as ENV variable instead of args in the docker-compose file.

Reply:
@butlerx any comments or suggestions?

Reply:
the problem with using build args is that is they are set at build time for the container which means users need to build their own containers.

one possible solution would be having the entryscript be 
```sh
!# /usr/bin/env sh
ssh-keyscan -H ${SSHHOST} >> ~/.ssh/known_hosts
yarn start $@
```
and set this as the entry point of the container